xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1613,11 +1607,6 @@ int CGUIWindowVideoBase::GetScraperForItem(CFileItem *item, ADDON::ScraperPtr &i
   return foundDirectly ? 1 : 0;
 }
 
-void CGUIWindowVideoBase::OnScan(const std::string& strPath, bool scanAll)
```

Comment:
Well I don't think that it's healthy to have one line methods. Didn't feels like it adds anything of value.